### DISCUSSION TITLE
Marking 4.1.0rc2 as Release

### DISCUSSION THREAD

**Discussion by borismod** [apache/superset#30574] [Q&A / Help] [answered]
Hello dear community,

The current latest version contains some vulnerabilities as a result of its third party libraries. It appears that these vulnerabilities were addressed in version 4.1.0rc2 according to Snyk report.
https://security.snyk.io/package/pip/apache-superset

Could you please mark the version as Release?

https://github.com/apache/superset/releases/tag/4.1.0rc2

Thanks a lot for your contribution.

**Comment by sfirke** [comment, marked as answer]
4.1.0rc2 is not a release, unfortunately.  It just got voted down on the dev email list serve.  If release candidate 3 (not yet created) passes its vote, it would become 4.1.0 which would then get marked as an official release.  There's a link on the wiki to the mailing list archive and instructions for how to join if you are interested in following the RC-to-release process: https://github.com/apache/superset/wiki
